borsh-derive/src/internals/attributes/item/mod.rs

```diff
@@ -284,8 +303,8 @@ mod tests {
         }
             })
         .unwrap();
-        let actual = check_item_attributes(&item_struct);
-        local_insta_assert_snapshot!(actual.unwrap_err().to_token_stream().to_string());
+        let actual = check_attributes(&item_struct);
+        local_insta_assert_debug_snapshot!(actual.unwrap_err());
     }
     #[test]
     fn test_init_function() {
```

Comment:
this test doesn't change due to [impl-ToTokens-for-Option](https://docs.rs/quote/1.0.33/quote/trait.ToTokens.html#impl-ToTokens-for-Option%3CT%3E)